storage/statedb/cache_redis_test.go

```diff
@@ -91,7 +91,6 @@ func TestRedisCache(t *testing.T) {
 
 	key, value := randBytes(32), randBytes(500)
 	cache.Set(key, value)
```

Comment:
I think all test codes should test `SetAsync`.
Currently, we only use `FastCache` or `HybridCache`. There is no case we use `remote.Set()` directly.

Reply:
@winnie-byun I added the same test cases using `SetAsync` instead of `Set`.

Reply:
How about `TestRedisCache_Timeout`?

Reply:
`SetAsync` finally calls `Set` and `Set` uses Redis client. The timeout function is only affected by the configuration of Redis client, I think testing `Set` is enough.